velox/serializers/PrestoSerializer.cpp

```diff
@@ -4220,12 +4220,19 @@ void PrestoVectorSerde::deserialize(
     auto compressBuf = folly::IOBuf::create(header.compressedSize);
     source->readBytes(compressBuf->writableData(), header.compressedSize);
     compressBuf->append(header.compressedSize);
+
+    // Process chained uncompressed results IOBufs.
```

Comment:
Hmm, we never run into this issue in production? Or we haven't turn on compression? I think compression is at least enabled for spill.

Reply:
single prod buffer is small enough to not trigger this bug

Reply:
What's the size to trigger this? Can you mention in PR description? Thanks!